### DISCUSSION TITLE
Configurable Tone of Voice

### DISCUSSION THREAD

**Discussion by azriel46d** [cheshire-cat-ai/core#607] [Ideas]
As much as it is interesting to have a whimisical cat giving the answers, not everyone appreciates and wants this kind of art .  :)

So, with no offence to the cat, why not make the `MAIN_PROMPT_PREFIX` configurable?

**Comment by pieroit** [comment]
@azriel46d two options:

- change the system prompt via [C.A.T. plugin](https://github.com/Furrmidable-Crew/cat_advanced_tools) by @nicola-corbellini. You can install it directly from the Plugins page in the admin, and it lets you edit the prompt directly into the plugin settings

- follow [this tutorial](https://cheshirecat.ai/write-your-first-plugin/) and use the `agent_prompt_prefix` hook in your own plugin
